src/components/Section/Section.md

```diff
@@ -0,0 +1,19 @@
+# Section
+
+Layout component (DOM wrapper) for more granular page components; allows for consistent styling of large 'sections' of the DOM.
+
+## When to use
+
+A Section can be used multiple times within a page/layout.
+
+## How to use
+
+A Section must contain child components, which the section will wrap in either `<div>` or `<section>`, depending on the `type` prop.
+
+## Accessibility notes
```

Comment:
I'm guessing this was copied from Header.md?

Reply:
Ah, thanks for flagging - fixed in new commit.